srcpkgs/fwupd/template

```diff
@@ -5,15 +5,14 @@ revision=1
 build_style=meson
 # manpages fail to build
 # tests require unpackaged umockdev
-configure_args="-Dbootdir=/boot/EFI -Dconsolekit=false -Dgtkdoc=false
```

Comment:
bootdir disappeared and never came back ?

Reply:
Yeah.Turns out it's not used by fwupd at all, was removed somewhere along the way. `find . -type f|xargs grep bootdir` shows nothing in their tree.

Reply:
All `-Dfoo` options also have to be present in `meson_options.txt`, you can see there is none: https://github.com/hughsie/fwupd/blob/master/meson_options.txt It's the same thing in the release archive, I checked.